src/common/circle-series.component.ts

```diff
@@ -133,19 +133,45 @@ export class CircleSeriesComponent implements OnChanges {
           barVisible: false,
           gradientId,
           gradientFill,
-          gradientStops: this.getGradientStops(color)
+          gradientStops: this.getGradientStops(color),
+          min: d.min,
+          max: d.max
         };
       }
     }).filter((circle) => circle !== undefined);
   }
 
-  getTooltipText({ tooltipLabel, value, seriesName }): string {
+  getTooltipText({ tooltipLabel, value, seriesName }, min: any, max: any): string {
     return `
       <span class="tooltip-label">${seriesName} • ${tooltipLabel}</span>
-      <span class="tooltip-val">${value.toLocaleString()}</span>
+      <span class="tooltip-val">${value.toLocaleString()}${this.getTooltipMinMaxText(min, max)}</span>
     `;
   }
 
+  getTooltipMinMaxText(min: any, max: any) {
+    if (min || max) {
```

Comment:
This should compare the values to `undefined` instead of truthy value here. A value of 0 would make this statement false.